src/en/reference-hook-tools.md

```diff
@@ -296,6 +296,28 @@ open-port 80/tcp
 open-port 1234/udp
 ```
 
+## payload-status-set
+
+`payload-status-set` is used to update the current status of a registered payload.
+The `class` and `id` provided must match a payload that
+has been previously registered with juju using [payload-register](#payload-register). The <status> must be one of the following:
```

Comment:
You can not use less-than and greater-than symbols in markdown like that. When rendered I do not see "&lt;status&gt;"